README.md

```diff
@@ -32,3 +32,18 @@ In `model-service` you had to choose `<ip-arg>` as well, this argument serves as
 Hence, in `.env` please set `MODEL_SERVICE_URL=http://<model-service-ip-arg>:5000/predict` e.g. `MODEL_SERVICE_URL=http://172.0.0.4:5000/predict`.
 
 
+# Vagrant
+
+NOTE: unsure if it works for arm, cause I cannot test it
+Make sure you have vagrant and VirtualBox/VMWare installed.
+Navigate to the vagrantfile for ARM or x86(normal) and run `vagrant up` in terminal. It will create 1 controller node and 2 worker nodes.
```

Comment:
```suggestion
Navigate to the vagrantfile for ARM or x86 (normal) and run `vagrant up` in terminal. It will create 1 controller node and 2 worker nodes.
```